odoo_initializer/tests/resources/config/initz_config_file.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Not sure what this file is about.
Isn't there already a test resource for this? [Here](https://github.com/mekomsolutions/odoo-initializer/tree/master/odoo_initializer/tests/resources/config)

Additionnaly the name of the file is not correct. Should be `iniz` not `initz`

Reply:
This is file is not useful, will delete it